### ISSUE TITLE
NVDA replicates itself upon subsequent runs

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#220]
**Reported by vtsaran on 16 Jul 2008 22:40 UTC**
Currently, if NVDA is running, and the user activates another copy of the program from the desktop, the second copy is launched. In fact, one can run three or more copies of NVDA at the same time.

Proposed behavior:
If NVDA is already running and the user activates an "NVDA" shortcut or presses ENTER on a desktop icon, NVDA main menu should be launched.
This will be helpful in cases where the user, for whatever reason, does not have access to the INSERT key and thus is not able to modify any of the NVDA configuration options.

**Comment by nvdaTrac** [comment]
**Comment by jteh on 16 Jul 2008 23:20 UTC**
What revision of NVDA are you running? This is a regression which was introduced in r2254, but was fixed in r2259. Does it occur beyond r2259 for you?

Your desired behaviour of bringing up the NVDA menu when activated again is not possible at this point. This should perhaps be filed as a separate enhancement request, but this will probably not occur any time soon.

**Comment by nvdaTrac** [comment]
**Comment by vtsaran on 17 Jul 2008 21:46 UTC**
Yes, this is working fine now.
I saw Mick's commit after I opened this ticket.
